docs/source/starter/installation.rst

```diff
@@ -67,16 +67,20 @@ If desired, the GeoIPS environment variables can be added to your
       # wget https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh
       wget https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-x86_64.sh
 
-- Make the install script executable and run the installer,
-  following the prompts (particularly the bit about
-  conda init / restarting terminal!):
+- Make the installation script executable, and run the installer.
+  Accept the license terms and cofirm the location.
+- You would see a message from installer asking to close and
+  re-open the shell at the end of installation which is not required.
 
   .. code:: bash
 
       chmod u+x ./Mambaforge-Linux-x86_64.sh
-      # After installing, follow instructions regarding conda init / restarting your terminal !
+      # No terminal reconnect is required afterward.
```

Comment:
Can this line be removed?

Reply:
I wanted to emphasize accommodating users who would focus solely on copying and pasting the required commands from the code blocks.

Reply:
It does not refer to the old comment. Line 77 is the old comment that has been deleted, as restarting the terminal is no longer required. Line 78 is the new one.

Reply:
Yes - why would someone expect that a terminal reset is required after running `chmod`?